Edited graphic

AB4B4B opened the pull request:
Spotted attribute errors in graphic. Graphic has been updated.

mikera commented:
Do you mean attribution? We should use free / open source graphics if we use anything I think.

AB4B4B commented:
> Do you mean attribution? We should use free / open source graphics if we use anything I think.

On the transformed dataset, I changed:
generatedBy -> wasGeneratedBy
derivedBy -> wasDerivedFrom

I also added author credits. Graphics are sourced from the Noun Project, with a Creative Commons license. Source and the creator's stated in the bottom of graphic
